src/server/set_family.h

```diff
@@ -33,6 +33,9 @@ class SetFamily {
   static int32_t FieldExpireTime(const DbContext& db_context, const PrimeValue& pv,
                                  std::string_view field);
 
+  static std::vector<long> SetFieldsExpireTime(const OpArgs& op_args, PrimeValue& pv,
```

Comment:
our style guide requires that we pass output variables (pv) last and by pointer.